npm/vite-dev-server/cypress/e2e/vite-dev-server.cy.ts

```diff
@@ -3,6 +3,58 @@
 import dedent from 'dedent'
 
 describe('Config options', () => {
+  it('works with tailwind', () => {
+    cy.scaffoldProject('tailwind-vite')
+    cy.openProject('tailwind-vite')
+    cy.startAppServer('component')
+
+    cy.visitApp()
+    cy.contains('App.cy.jsx').click()
+    cy.waitForSpecToFinish()
+    cy.get('.passed > .num').should('contain', 1)
+    cy.withCtx(async (ctx) => {
+      await ctx.actions.file.writeFileInProject(
+        'src/App.jsx', `
+        import React from 'react'
+        export const App = () => {
+          return (
+            <div className='bg-blue-100'>
+              Hello
+            </div>
+          )
+        }
+        `,
+      )
```

Comment:
Why is this needed if you're only using a component defined within the the spec?

Reply:
Oops, it's not. I'll remove it.

Reply:
Done